models/modules/unet_generator_attn/video_unet_generator_attn.py

```diff
@@ -0,0 +1,1391 @@
+                                    increased efficiency.
+    """
+
+    def __init__(
+        self,
+        image_size,
+        in_channel,
+        inner_channel,
+        out_channel,
+        res_blocks,
+        attn_res,
+        tanh,
+        n_timestep_train,
+        n_timestep_test,
+        norm,
+        group_norm_size,
+        cond_embed_dim,
+        dropout=0,
+        channel_mults=(1, 2, 4, 8),
+        conv_resample=True,
+        use_checkpoint=False,
+        use_fp16=False,
+        num_heads=1,
+        num_head_channels=-1,
+        num_heads_upsample=-1,
+        use_scale_shift_norm=True,
+        resblock_updown=True,
+        use_new_attention_order=True,  # False,
+        efficient=False,
+        freq_space=False,
+    ):
+        super().__init__()
+
+        if num_heads_upsample == -1:
+            num_heads_upsample = num_heads
+
+        self.image_size = image_size
+        self.in_channel = in_channel
+        self.inner_channel = inner_channel
+        self.out_channel = out_channel
+        self.res_blocks = res_blocks
+        self.attn_res = attn_res
+        self.dropout = dropout
+        self.zero_dropout = 0.0
+        self.channel_mults = channel_mults
+        self.conv_resample = conv_resample
+        self.use_checkpoint = use_checkpoint
+        self.dtype = torch.float16 if use_fp16 else torch.float32
+        self.num_heads = num_heads
+        self.num_head_channels = num_head_channels
+        self.num_heads_upsample = num_heads_upsample
+        self.freq_space = freq_space
+
+        if self.freq_space:
+            from ..freq_utils import InverseHaarTransform, HaarTransform
+
+            self.iwt = InverseHaarTransform(3)
+            self.dwt = HaarTransform(3)
+            in_channel *= 4
+            out_channel *= 4
+
+        if norm == "groupnorm":
+            norm = norm + str(group_norm_size)
+
+        self.cond_embed_dim = cond_embed_dim
+
+        ch = input_ch = int(channel_mults[0] * self.inner_channel)
+        self.input_blocks = nn.ModuleList(
+            [EmbedSequential(InflatedConv3d(in_channel, ch, 3, padding=1))]
+        )
+        self._feature_size = ch
+        input_block_chans = [ch]
+        ds = 1
+        for level, mult in enumerate(channel_mults):
+            for _ in range(res_blocks[level]):
+                layers = [
+                    ResBlock(
+                        ch,
+                        self.cond_embed_dim,
+                        self.zero_dropout,
+                        out_channel=int(mult * self.inner_channel),
+                        use_checkpoint=use_checkpoint,
+                        use_scale_shift_norm=use_scale_shift_norm,
+                        norm=norm,
+                        efficient=efficient,
+                        freq_space=self.freq_space,
+                    )
+                ]
+                ch = int(mult * self.inner_channel)
+                if ds in attn_res:
```

Comment:
Variable `attn_res` should not condition the motion module (MM). The MM is mandatory, not conditioned I believe. 

Also, `attn_res` conditions the `AttentionBlock` in the frame-only UNet, and we should keep this code here as well.

This is because the MM is an addition to any configuration of the frame-only UNet.